OpenOversight/app/main/views.py

```diff
@@ -288,10 +288,16 @@ def get_tutorial():
 @login_required
 def profile(username: str):
     if re.search("^[A-Za-z][A-Za-z0-9_.]*$", username):
-        user = User.by_username(username).one()
+        try:
+            user = User.by_username(username).one()
+        except NoResultFound:
+            abort(HTTPStatus.NOT_FOUND)
```

Comment:
Without this error check the route would error out upon entering an invalid `username`.

Reply:
Good catch!